controllers/articles.controllers.js

```diff
@@ -52,9 +50,30 @@ const postCommentForArticle = (req, res, next) => {
     });
 };
 
+const patchVotesForArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { upd_votes } = req.body;
+
+  if (typeof upd_votes !== "number") {
+    return next({
+      status: 400,
+      msg: "Invalid input: 'upd_votes' must be a number",
+    });
+  }
```

Comment:
You don't need to manually check this, let psql try and use the value and it will return an error. Then you can catch the err.code and return an error based on that. That way, if you ever come across the same code you have already dealt with it